### PR TITLE
Feature/link database tanslator sections

### PR DESCRIPTION
**Pull request by edoulaX**

### CODE REVIEW

File: app/src/main/java/com/github/sdpsharelook/Section/SectionDetail.kt [hunk 1]
```diff
@@ -18,6 +21,37 @@ class SectionDetail : AppCompatActivity() {
...
+//                                text = message
+//                            }
+//                        }
+//                    }
+//                    it.isFailure -> {
+//                        it.exceptionOrNull()?.printStackTrace()
+//                    }
+//                }
+//            }
+//        }
```
Comment (villemontc) [inline app/src/main/java/com/github/sdpsharelook/Section/SectionDetail.kt:40]: Is this voluntary those comments ?

File: app/src/main/java/com/github/sdpsharelook/Section/SectionDetail.kt [hunk 2]
```diff
@@ -18,6 +21,37 @@ class SectionDetail : AppCompatActivity() {
...
+        if(addWordToSection){
+            val wordTranslated = intent.getSerializableExtra(TRANSLATOR_WORD) as SectionWord
+            CoroutineScope(Dispatchers.IO).launch {
+                if (section != null){
+                    section.databaseRepo.insert(section.sectionRepo, wordTranslated.toList())
+                }
+            }
+            addSectionWord(wordTranslated)
+            addWordToSection = false
+        }
```
Comment (villemontc) [inline app/src/main/java/com/github/sdpsharelook/Section/SectionDetail.kt:53]: This branch is not tested

**Review by villemontc — APPROVED** [review]
Ok but you could improve those tests to match 80% coverage at least

**Review by albastardoto — APPROVED** [review]
Approved but still needs tests !!!

**Pull request merged**